### ISSUE TITLE
Truncate Dims

### ISSUE THREAD

**Issue by tobiasschweizer** [Snowflake-Labs/arctic-embed#8]
Hi,

I've read about truncating an embedding's dims in your [README](https://github.com/Snowflake-Labs/arctic-embed?tab=readme-ov-file#using-sentence-transformers) using `torch.nn.functional.normalize`.

Is the results the same when using sentence transformer's `truncate_dim` [param](https://sbert.net/examples/training/matryoshka/README.html#inference)? 

Thanks
